docs/getting-started/prerequisites.md

```diff
@@ -1,18 +1,33 @@
-# Prerequisites
+## 🛠 Prerequisites
 
-1. **Install `uv` 🚀**
+Before setting up the project, make sure the following tools are installed on your system:
 
-    Uv is used as a python package manager and project manager. You can install it following the instructions in the official documentation:
+### 1. 🐍 Install [Python 3.10+](https://www.python.org/downloads/)
```

Comment:
I would suggest installing python via `uv`, `uv python install`, see https://docs.astral.sh/uv/guides/install-python/. Or at least use the `.python-version` file